tools/pika_cdc/README.md

```
@@ -0,0 +1,24 @@
+```bash
+brew install protobuf
+go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
+```
+
+## Build pika cdc
+```bash
+make
+```
+
+## Todo:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)